src/Storefront/Resources/app/storefront/src/plugin/main-menu/offcanvas-menu.plugin.js

```diff
@@ -47,8 +53,8 @@ export default class OffcanvasMenuPlugin extends Plugin {
      * @private
      */
     _registerEvents() {
-        this.el.removeEventListener(this.options.tiggerEvent, this._getLinkEventHandler.bind(this));
-        this.el.addEventListener(this.options.tiggerEvent, this._getLinkEventHandler.bind(this));
+        this.el.removeEventListener(this.options.triggerEvent, this._getLinkEventHandler.bind(this));
```

Comment:
Isn't this already kind of a break? When I still use the old/deprecated "tigger" 😆 it will have no effect because the new one is already used.

Reply:
ah no, I did this trick in init

```
        /** @deprecated tag:v6.7.0 - Remove if condition completely, as "tiggerEvent" will be removed */
        if (this.options.tiggerEvent !== this.options.triggerEvent) {
            this.options.triggerEvent = this.options.tiggerEvent;
        }
```

that should work, right?

Reply:
That should work, or you can use the feature flag and have it more verbose.

```js
/** @deprecated tag:v6.7.0 - Remove "event" completely, as "tiggerEvent" will be removed */
const event = window.Feature('v6.7.0') ? this.options.triggerEvent : this.options.tiggerEvent;

this.el.removeEventListener(event, this._getLinkEventHandler.bind(this));

```